### ISSUE TITLE
Updating HighCharts with Ajax Issue

### ISSUE THREAD

**Issue by ecreamer** [highcharts/highcharts-react#307]
I am having issues updating my HighCharts data after an Ajax call. My Ajax file is sending new data to my page, and the data is arriving correctly. However, using the update function for the chart is not doing anything. Here are my updates:

$('#ChartContainer').highcharts().series[0].update({
	data: data
});

$('#ChartContainer').highcharts().xAxis[0].update({
	categories: labels
});

For reference, "data" and "labels" are arrays of the data that has been sent via Ajax. "ChartContainer" is the ID where my chart lies. We are using HighChart 9.1.0. This exact code worked fine when using 7.2.0, but upon update this no longer works.
